NickvisionCavalier.Shared/Controllers/MainWindowController.cs

```diff
@@ -45,18 +44,54 @@ public class MainWindowController
     /// </summary>
     public bool AutohideHeader => Configuration.Current.AutohideHeader;
 
+    private readonly PreferencesViewController _preferencesViewController;
```

Comment:
Turn this into a public get, init property.

```
public PreferencesViewController PreferencesViewController { get; init; }
```

Then you can also remove the `GetPreferencesViewController()` method.